mapbox/mapbox-gl-native
# Render Extrusion Test

This PR contains a test that rotates around an extrusion to manually detect render artefacts. For ease of use, I removed all additional tests in the project, so you don't need to setup a test filter in AWS Device farm.

![ezgif com-video-to-gif 16](https://cloud.githubusercontent.com/assets/2151639/25407866/4981a246-2a0c-11e7-8223-faa557dc8ef1.gif)


Steps to run tests:
- Cherry pick commit from this PR
- execute `make android-ui-test-arm-v7`
- login to AWS
- Open Device Farm
- open `Mapbox GL Android` 
- click `Create a new run`
- click upload and select `platform/android/MapboxGLAndroidSDKTestApp/build/outputs/apk/MapboxGLAndroidSDKTestApp-debug.apk`, next step
- configure test, select `Instrumentation`, click upload `platform/android/MapboxGLAndroidSDKTestApp/build/outputs/apk/MapboxGLAndroidSDKTestApp-debug-androidTest.apk`, next step
- device pool, `ASUS Nexus 7 - Android 4.2.1`, next step, next step, confirm and start run

Please do not merge this PR, solely for testing purposes.
